packages/editor/src/components/post-author/style.scss

```diff
@@ -0,0 +1,100 @@
+.autocomplete__wrapper {
```

Comment:
This is not a very unique class name. I see they provide the option to customize a namespace:

https://github.com/alphagov/accessible-autocomplete#cssnamespace-default-autocomplete

Could we use this to bring within respect of our own coding guidelines?

https://github.com/WordPress/gutenberg/blob/master/docs/contributors/coding-guidelines.md#naming

i.e. `components-post-author__autocomplete`

That might risk creating some invalid names, since they'd append their own `__whatever` for BEM syntax. Alternatively, this is the sort of thing where we'd create our own component, not use directly within PostAuthor, so that it could be defined as `components-autocomplete`.

Then we run into the issue of... we already an Autocomplete, which opens up a whole other can of worms about what it is that makes them distinct (an interesting question to pose!).

Reply:
> Could we use this to bring within respect of our own coding guidelines?

Sure, I  can change the prefix to `components-post-author__autocomplete` and adjust the css file.

Reply:
> Then we run into the issue of... we already an Autocomplete, which opens up a whole other can of worms about what it is that makes them distinct (an interesting question to pose!).

Despite the naming and overall similar functionality, these are quite different components when you examine the details. The existing Autocomplete component is designed to be used inline, inside the rich text editor - I explored using it for the post author selector in https://github.com/WordPress/gutenberg/pull/5921 and although it sort of worked, there were some big issues.

This comment and the responses do a good job clarifying the accessibility issues we are trying to address here: https://github.com/WordPress/gutenberg/pull/5921#issuecomment-383290782 and why we need something beyond what our existing Autocomplete component offers.

Reply:
addressed css prexif naming in https://github.com/WordPress/gutenberg/pull/7385/commits/807b61efe5f08cc509c927fccdc78e99a32d1098